src/execution_plan/ops/op_create.h

```diff
@@ -15,6 +15,13 @@
 
 /* Creates new entities according to the CREATE clause. */
 
+// Container struct for properties to be added to a new graph entity
+typedef struct {
```

Comment:
What's wrong with PropertyMap?

Reply:
This stores SIValues that have been converted from the AR_ExpNodes of PropertyMap. We could use the PropertyMap struct by calling `AR_EXP_NewConstOperandNode` on each value, but I think that would be more wasteful than necessary.